src/views/v2/rsvp/actions/success.php

```diff
@@ -39,9 +35,26 @@ class="tribe-common-form-control-toggle__input tribe-tickets__rsvp-actions-succe
 			class="tribe-common-form-control-toggle__label tribe-tickets__rsvp-actions-success-going-toggle-label"
 			for="<?php echo esc_attr( $toggle_id ); ?>"
 		>
-			<?php esc_html_e( 'Show me on public attendee list', 'event-tickets' ); ?>
+			<span
+				data-js="tribe-tickets-tooltip"
+				data-tooltip-content="#tribe-tickets-tooltip-content-<?php echo esc_attr( $rsvp->ID ); ?>"
+				aria-describedby="tribe-tickets-tooltip-content-<?php echo esc_attr( $rsvp->ID ); ?>"
+			><?php
```

Comment:
:no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: 984a8d11f115c1ba5ddacd81dd16ac61</sub>